audit-resolve.json

```diff
@@ -4,8 +4,16 @@
       "decision": "ignore",
       "madeAt": 1626802771020,
       "expiresAt": 1640995200000
+    },
+    "1091239|opensearch-browser>isomorphic-fetch>node-fetch": {
+      "decision": "ignore",
+      "madeAt": 1692723196646
+    },
+    "1091791|opensearch-browser>isomorphic-fetch>node-fetch": {
+      "decision": "ignore",
+      "madeAt": 1692723202247
     }
   },
   "rules": {},
   "version": 1
-}
+}
```

Comment:
Should there be a newline to end this file?  (Maybe there is, but the icon at the end makes me think that there isn't one?  Not sure what that means).

Reply:
Good question. I'll take a look.